osu.Game/Database/BeatmapDatabase.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using osu.Framework.OS;
+using SQLite;
+
+namespace osu.Game.Database
+{
+    public class BeatmapDatabase
+    {
+        private static SQLiteConnection Connection { get; set; }
```

Comment:
No, you can't have the same beatmap file open multiple times. It's also not thread safe.